reducers/views/rhs_suppressed.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {ViewsState} from 'types/store/views';
+import {ActionTypes} from 'utils/constants';
+
+export default function rhsStash(state: ViewsState['rhsSuppressed'] = false, action: GenericAction): boolean {
```

Comment:
Should we rename this to be `rhsSuppressed` vs. `rhsStash`?

Reply:
thanks for noticing! will do